core/rawdb/accessors_chain.go

```diff
@@ -801,8 +801,9 @@ func WriteBlock(db ethdb.KeyValueWriter, block *types.Block) {
 // WriteAncientBlocks writes entire block data into ancient store and returns the total written size.
 func WriteAncientBlocks(db ethdb.AncientWriter, blocks []*types.Block, receipts []types.Receipts, td *big.Int) (int64, error) {
 	var (
-		tdSum      = new(big.Int).Set(td)
-		stReceipts []*types.ReceiptForStorage
+		blobSidecarFound = false
```

Comment:
Why is `blobSidecarFound` needed? The function in `ModifyAncients` seems to be only called once so I wonder when the second time `blobSidecarFound` will need to be checked.